frontend/src/components/Header/Header.tsx

```diff
@@ -152,6 +152,26 @@ export default function Header() {
                   </Link>
                 )}
               </li>
+              <li className="nav-wall">
+                {signed ? (
+                  null
+                ) : (
+                  <div className="nav-wall vr text-black"></div>
+                )}
+              </li>
+              <li className="nav-item">
+                {signed ? (
+                  null
+                ) : (
+                  <Link
```

Comment:
Ajustar para o link para inscrição ficar ao lado da opção de início e não depois do login no header 

![image](https://github.com/user-attachments/assets/0a1c74e4-d7a8-49b9-817f-db4140278320)

Reply:
concordo